### DISCUSSION TITLE
Welcome to HomeAssistant_OneControl_Discovery Discussions!

### DISCUSSION THREAD

**Discussion by rperciaccante** [rperciaccante/HomeAssistant_OneControl_Discovery#16] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by smhogan** [comment]
Thank You for this excellent integration. It's clear you have spent some time on this.  There are a few items that are listed in cloud gateway that don't seem to get pulled down to HA. Is there a way to import these? Also, does anyone have any experience in unlocking devices in OneControl? For example, awning lights and soffit lights. These do not pull down during the export.  Also, has there been any development in RGB lights being imported?
I have a 2024 Brinkley G4000 if you would be interested in looking at this data for future advancements.

**Reply by simpskvi** [reply]
I actually removed the RGB lights from the excluded list in Node Red and ran discovery again. I can turn them on and off now, but I am unable to adjust the color. Not a problem for me since I usually leave them the same colors anyways.

**Comment by MeAndMyShadows** [comment]
I’ve been using this for about a year now in my 2021 Grand Design Momentum 397TH-R. Lately, I have been unable to change the Heat Source through Home Assistant. Changes to that setting through OneControl are immediately reflected in HA, so I know that setting is being communicated between the two systems. Is there something I need to update in order to change this setting in HA? In addition, the temperature I have set doesn’t always match up in both systems. 
![IMG_0844](https://github.com/rperciaccante/HomeAssistant_OneControl_Discovery/assets/123312722/1ac74f6a-ffb7-4609-8774-95903a06552d)
![IMG_0843](https://github.com/rperciaccante/HomeAssistant_OneControl_Discovery/assets/123312722/eedf48d5-064f-44ce-92c5-089bc3a9869e)


Thanks!

**Reply by simpskvi** [reply]
I'm having the same issue in my 2022 Momentum 397TH-R. I tried to figure it out on my own, but finally gave up.

**Reply by rperciaccante** [reply]
I no longer have my Momentum (I traded it in for a Solitude) so I can no longer fix this on my own, however, if someone gives me access, I can help remotely.

**Reply by MeAndMyShadows** [reply]
How do I give you remote access?

**Comment by rusvasul** [comment]
This is a great integration. I have set it up on my 2022 Grandesign momentum 395ms and it is working great! thank you for your contribution.

**Reply by rumlak** [reply]
Bob is no longer supporting this project and there is a new project that no longer requires the cloud gateway.  It connects directly to the bluetooth in the OC board.  The integration reports back more entities than this one did.

https://github.com/phurth/ha-onecontrol
